### PR TITLE
GS01SUMO_Deploy.csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added CGCON-NCPM02-00013 to SUMO-4 deployment and updated CUIDs:

Inserted the following into deployment 4 (extra CUSTARD CPM) - line 154
DY096,,DY112,,GS01SUMO-RIC21-00-CPMENG000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGCON-NCPM02-00013,-54.407168,-89.206037,,12,4589,

Also updated the CUIDs to match recent alignment with R2R CUID formats.  Looks like all lines are highlighted again, but the only changes on deployments 1-3 are to the CUIDs.

### CODE REVIEW

File: deployment/GS01SUMO_Deploy.csv [hunk 1]
```diff
@@ -160,6 +160,7 @@ DY096,,DY112,,GS01SUMO-RID16-05-PCO2WB000,4,1,2018-12-04T17:10:40,2020-01-20T10:
 DY096,,DY112,,GS01SUMO-RID16-06-DOSTAD000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-DOSTAD-00507,-54.407168,-89.206037,,12,4589,
 DY096,,DY112,,GS01SUMO-RID16-07-NUTNRB000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-NUTNRB-01103,-54.407168,-89.206037,,12,4589,
 DY096,,DY112,,GS01SUMO-RID16-08-SPKIRB000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-SPKIRB-00291,-54.407168,-89.206037,,12,4589,
+DY096,,DY112,,GS01SUMO-RIC21-00-CPMENG000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGCON-NCPM02-00013,-54.407168,-89.206037,,12,4589,
```
Comment (spetillo) [inline deployment/GS01SUMO_Deploy.csv:163]: To be consistent with other stuff added for NOC, add Note:
"Extra CPM for NOC deployment"

**Review by spetillo — CHANGES_REQUESTED** [review]
One comment to add a note that the extra CPM is for NOC, otherwise all changes of consequence look good!

File: deployment/GS01SUMO_Deploy.csv [hunk 2]
```diff
@@ -160,6 +160,7 @@ DY096,,DY112,,GS01SUMO-RID16-05-PCO2WB000,4,1,2018-12-04T17:10:40,2020-01-20T10:
...
 DY096,,DY112,,GS01SUMO-RID16-07-NUTNRB000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-NUTNRB-01103,-54.407168,-89.206037,,12,4589,
 DY096,,DY112,,GS01SUMO-RID16-08-SPKIRB000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-SPKIRB-00291,-54.407168,-89.206037,,12,4589,
+DY096,,DY112,,GS01SUMO-RIC21-00-CPMENG000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGCON-NCPM02-00013,-54.407168,-89.206037,,12,4589,
 DY096,,DY112,,GS01SUMO-RID27-00-DCLENG000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGCON-NDCL27-00013,-54.407168,-89.206037,,12,4589,Extra DCL for NOC deployment
 DY096,,DY112,,GS01SUMO-RID27-01-NOCNIA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,PICGN-NOCNIA-0N112,-54.407168,-89.206037,,12,4589,Extra sensor for NOC deployment
 DY096,,DY112,,GS01SUMO-RID27-02-NOCSIA000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,PICGN-NOCSIA-Si009,-54.407168,-89.206037,,12,4589,Extra sensor for NOC deployment
```
Comment (batryn) [inline deployment/GS01SUMO_Deploy.csv:166]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

File: deployment/GS01SUMO_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/GS01SUMO_Deploy.csv]: comment added!

File: deployment/GS01SUMO_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/GS01SUMO_Deploy.csv:166]: Looks good.

**Review by snwhite — APPROVED** [review]

**Review by spetillo — APPROVED** [review]
Great!

**Pull request merged**